R/S3.R

```diff
@@ -967,11 +973,13 @@ pander.CrossTable <- function(x, caption = attr(x, 'caption'), digits = panderOp
         res <- rbind(res, res.r)
     }
     res.r <- NULL
-    for (j in 1:nc) {
-        res.r <- cbind(res.r, paste(nt[ (nt.nr - ts + 1) : nt.nr, j], collapse = "\\ \n"))
+    if (total.c) {
+        for (j in 1:nc) {
+            res.r <- cbind(res.r, paste(nt[ (nt.nr - ts + 1) : nt.nr, j], collapse = "\\ \n"))
+        }
+        res <- rbind(res, res.r)
     }
-    res <- rbind(res, res.r)
-    cln <- c(if (RowData != "") RowData else '&nbsp;', colnames(x$t), 'Total')
+    cln <- c(if (RowData != "") RowData else '&nbsp;', colnames(x$t), if (total.r) 'Total')
```

Comment:
I try to avoid using `ifelse`, since it requires branches to be of equal length (otherwise, it just populates the length). But you are definitely right in this case, since `RowData` is always a string